dnsapi/dns_variomedia.sh

```diff
@@ -93,11 +93,11 @@ dns_variomedia_rm() {
 # _sub_domain=_acme-challenge.www
 # _domain=domain.com
 _get_root() {
-  fulldomain=$1
-  i=1
+  domain=$1
+  i=2
```

Comment:
i  should start from "1"
There is no "_acme-challenge" prefix if it's working in dns alias mode.

Reply:
Ok I changed that, Github actions tests successfully completed.